### PR TITLE
update btc max send /utxo max send

### LINKED ISSUE #409: Send form UI Update
## Description

3 send forms 
evM
utxo 
cosmos
Screenshot 2024-10-11 at 3 19 55 PM

dark mode on address select isn't amazing
Screenshot 2024-10-11 at 3 11 14 PM

the input fields need to be modernised, rounded edges

### LINKED ISSUE #434: BTC Ledger: Cannot send max balance to keystore wallet.
## TODO

- TODO 1
- TODO 2
- ...

### PR DESCRIPTION
**Pull request by Thorian1te**

### CODE REVIEW

File: src/renderer/services/bitcoin/transaction.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const createTransactionService = (client$: Client$, network$: Network$):
       amount,
       sender,
       feeRate,
-      feeOption: undefined,
+      feeOption: feeOption,
```
Comment (cinnamoroll6130) [inline src/renderer/services/bitcoin/transaction.ts]: We don't need to assign the value here.

Just `feeOption,` would be okay.

**Review by cinnamoroll6130 — COMMENTED** [review]
lgtm.

File: src/main/api/ledger/bitcoin/transaction.ts [hunk 2]
```diff
@@ -85,7 +85,31 @@ export const send = async ({
...
+    const fee = await clientLedger.getFeesWithRates({ sender, memo })
+    const feeAmount = fee.fees[feeOption]
+    const feeRate = fee.rates[feeOption]
+    const bal = await clientLedger.getBalance(sender)
+    const btcBalance = bal[0]
+    const transactionSize = feeAmount.amount().toNumber() / feeRate
+    const roundedFeeRate = Math.ceil(feeRate)
+    const adjustedFee = baseAmount(roundedFeeRate * transactionSize)
+    const feeValue = adjustedFee.amount().toNumber()
+    const roundedFeeValue = Math.ceil(feeValue / 1000) * 1000
```
Comment (GrassRootsCrypto) [inline src/main/api/ledger/bitcoin/transaction.ts:98]: Why is this line here and not in BCH below? I assume they should be the same, as with other UTXOs like LTC.

File: src/renderer/services/bitcoin/transaction.ts (hunk 1 above)
Comment (Thorian1te) [reply src/renderer/services/bitcoin/transaction.ts]: good catch

File: src/main/api/ledger/bitcoin/transaction.ts (hunk 2 above)
Comment (Thorian1te) [reply src/main/api/ledger/bitcoin/transaction.ts:98]: cus with bch I don't need to round the fee up.

**Review by cinnamoroll6130 — DISMISSED** [review]
looks good

**Review by cinnamoroll6130 — APPROVED** [review]
